src/_includes/layouts/membership.liquid

```diff
@@ -2,29 +2,87 @@
 layout: layouts/base.liquid
 ---
 
-<section class="desktop:p-0">
-  <div class="w-full z-[-1] ml-auto tablet:h-[75%] 2xl:h-[90%] desktop:ml-0 desktop:absolute tablet:w-[45%] desktop:w-full">
-    <img src="/assets/img/bg-triangle-pink3.png" alt="" class="h-[21rem] ml-auto w-auto object-contain lg:max-w-[50%] tablet:h-auto tablet:float-right desktop:h-full">
+<section>
+  <div class="w-full z-[-1] ml-auto tablet:h-[75%] 2xl:h-[50%] desktop:ml-0  tablet:w-[45%] desktop:w-full">
+    <img
+      src="/assets/img/membershipTopPhoto.png"
+      alt=""
+      class="h-[15rem] ml-auto w-auto object-contain lg:max-w-[50%] tablet:h-auto tablet:float-right desktop:w-[34%]"
+      style="z-index: 10; position: relative; margin-bottom: 2rem">
   </div>
-  <div class="container p-4 md:p-12 lg:mx-auto 2xl:max-w-[75rem]">
-    <div class="p-4 lg:p-12 md:max-w-[50%] lg:max-w-[60%]">
-      <h1 class="text-magenta leading-tight mb-12 md:my-12">{{ membershipHeader }}</h1>
-      {% if membershipAnnouncement != "" %}
-        <div class="bg-greenLight text-greenDark py-3 px-5 mb-6">
-          <h2 class="text-greenDark mb-1">Announcement</h2>
-          {{ membershipAnnouncement | markdownify }}
-        </div>
-      {% endif %}
+  <div class="container 2xl:max-w-[100%]" style="padding-bottom: 16rem;">
+    <div class="p-4 lg:p-8 lg:max-w-[100%]" style="position: absolute; background: #FAFAFF;">
```

Comment:
Yeah, try to leverage Tailwind CSS rather than explicitly injecting inline styling.